### ISSUE TITLE
BRD - Quest for Dark Iron Ore Smelting - Gloom'rel

### ISSUE THREAD

**Issue by scotty0100** [classicdb/database#263]
Map & Area: BRD - Summoner's Tomb

How to replicate the bug: speak to Gloom'rel 

What happens: nothing, you cannot hand in the required items

What should happen: you should be able to hand in the required items and get taught how to smelt dark iron

Source: 
http://db.vanillagaming.org/?npc=9037#wh-comments

**Comment by scotty0100** [comment]
Looked into this in a bit more detail. It does seem to be working already, just that the chalice was in the wrong position.

Run SQL Query: 

UPDATE gameobject SET position_x = '1226.689' , position_y = '-244.472' WHERE guid = 67871;

And then all should be working fine :)

Note: To test you need a miner + 240 skill to meet conditions of Gossip Option

**Comment by evil-at-wow** [comment]
UDB (and hence TBC-DB) also had wrong position data. I added a fix in TBC-DB 1.3.1 with a correct position coming from my own sniffs.

```
-- Add missing spawn for the Spectral Chalice, but initially despawned.
-- Talking to Gloom'rel with a 230+ miner that doesn't know "Dark Iron Smelting" should spawn the object.
-- Note: the data is taken from UDB, but their position is wrong. They spawn it next to Anger'rel.
--       Fixed position with my own data from sniffs (patch 3.1.3 data).
DELETE FROM gameobject WHERE id = 164869;
INSERT INTO gameobject (guid, id, map, spawnMask, position_x, position_y, position_z, orientation, rotation0, rotation1, rotation2, rotation3, spawntimesecs, animprogress, state) VALUES
(67871, 164869, 230, 1, 1224.6047, -244.5426, -85.0765, 2.26893, 0, 0, 0.906307, 0.422619, -250, 0, 1);
```

And I remember it being 230+ mining required, but I never played in Classic so it could be different there.
